### PR TITLE
clarify responsibilities for the person clicking "merge"

### PR DESCRIPTION
**Pull request by Ekleog-NEAR**

### CODE REVIEW

**Review by VanBarbascu — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -136,6 +136,13 @@ the existing super owners will discuss and approve the addition.  These
 discussions take place on zulip so if you are interested in becoming a
 code reviewer, please reach out to us there.
 
+The person who clicks the "Merge when ready" button is the one who
+guarantees that no unreviewed code was added between the approving review
+and the commit that is being landed. They must be especially careful of
+commits that might have happened before the review, but not have been
+reviewed (eg. because the review was already in progress), as they will
+show above the review line on the GitHub UI.
```
[the remark this answers is not in the document]
Comment (wacban) [reply CONTRIBUTING.md:144]: Maybe in the `## After the PR is submitted 3) ` section above you can point readers to this section? I'm worried someone may not read the whole thing and stop after reading how to merge PRs.

**Review by wacban — APPROVED** [review]
LGTM

**Comment by Ekleog-NEAR** [comment]
I’ve handled the review comments, will leave this open for a bit more time to see if other people have concerns :)

**Comment by Ekleog-NEAR** [comment]
Seems like no one else has any comments, let’s land this :)

**Pull request merged**